### PR TITLE
Add fur ban data

### PR DESCRIPTION
**Pull request by pabloarosado**
Add steps about data on laws banning fur farming and trading.

Additionally, I made some small changes in a function of etl.helpers, and fixed a bug in wizard.

### CODE REVIEW

**Review by lucasrodes — APPROVED** [review]
Nice work, appreciate the well documented code.

LGTM

File: etl/steps/data/meadow/animal_welfare/2023-09-08/fur_laws.py [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+        ("Cyprus", "YES", "NO", "", "", "YES"),
+        ("Czech Republic", "YES", "YES", "", "2019", "NO"),
+        # ("Denmark", "YES", "PARTIAL: ban on fox farming", "", "", "YES"),
+        # ("Denmark", "YES", "PARTIAL: phase-out of raccoon dog farming", "", "", "YES"),
+        # Combined:
+        ("Denmark", "YES", "PARTIAL", "", "", "YES"),
+        ("Estonia", "YES", "YES", "", "2026", "NO"),
+        ("Finland", "YES", "NO", "", "", "YES"),
+        ("France", "YES", "PARTIAL: non-domestic species ban", "", "November 2021", "NO"),
+        ("Germany", "YES", "YES: phased-out due to stricter regulations", "", "2022", "NO"),
```
[the remark this answers is not in the document]
Comment (pabloarosado) [reply etl/steps/data/meadow/animal_welfare/2023-09-08/fur_laws.py:64]: In the meadow step I was just copying the original content, following the same structure as the original table (which was pretty hard to parse, it was paragraphs on a table inside a multi-page pdf).

File: etl/steps/data/garden/animal_welfare/2023-09-08/fur_laws.py [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+}
+
+
+def prepare_fur_farming_ban_status(tb: Table) -> Table:
+    tb = tb.copy()
+    # Rename fur farming ban statuses.
+    # Find all status that start with "YES" and map them to "YES".
+    tb.loc[tb["fur_farming_status"].str.startswith("YES"), "fur_farming_status"] = "YES"
+    # Find all status that start with "PARTIAL" and map them to "PARTIAL".
+    tb.loc[tb["fur_farming_status"].str.startswith("PARTIAL"), "fur_farming_status"] = "PARTIAL"
```
[the remark this answers is not in the document]
Comment (pabloarosado) [reply etl/steps/data/garden/animal_welfare/2023-09-08/fur_laws.py:53]: I prefer the more explicit one.
